docs/dg/dev/third-party-module-development/develop.md

```diff
@@ -0,0 +1,159 @@
+
+class ProductCategoryProductTableConfigurationExpanderPlugin extends AbstractPlugin implements ProductTableConfigurationExpanderPluginInterface
+{
+    /**
+     * {@inheritDoc}
+     * - Expands `ProductTable` configuration with categories column.
+     *
+     * @api
+     *
+     * @param \Spryker\Zed\Gui\Communication\Table\TableConfiguration $config
+     *
+     * @return \Spryker\Zed\Gui\Communication\Table\TableConfiguration
+     */
+    public function expandTableConfiguration(TableConfiguration $config): TableConfiguration
+    {
+         // Let's insert `Categories` column right after the first column header (product ID)
+        $headers = $config->getHeader();
+        $firstColumnHeader = array_shift($headers);
+        array_unshift($headers, 'Categories');
+        array_unshift($headers, $firstColumnHeader);
+
+        $config->setHeader($headers);
+
+        return $config;
+    }
+}
+
+```
+
+{% info_block infoBox %}
+    Validate that on http://backoffice.de.spryker.local/product-management page you can see categories column in the table header.
+{% endinfo_block %}
+
+**c.** Create a plugin that will provide data for the **Categories** column
+
+
+File Path: vendor/your-company-name/product-category/src/YourCompanyName/Zed/ProductCategory/Communication/Plugin/ProductManagement/ProductCategoryProductTableDataBulkExpanderPlugin.php
+
+```php
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace YourCompanyName\Zed\ProductCategory\Communication\Plugin\ProductManagement;
+
+use Generated\Shared\Transfer\ProductCategoryConditionsTransfer;
+use Generated\Shared\Transfer\ProductCategoryCriteriaTransfer;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+use Spryker\Zed\ProductCategory\Business\ProductCategoryFacade;
+use Spryker\Zed\ProductManagementExtension\Dependency\Plugin\ProductTableDataBulkExpanderPluginInterface;
+
+class ProductCategoryProductTableDataBulkExpanderPlugin extends AbstractPlugin implements ProductTableDataBulkExpanderPluginInterface
+{
+    /**
+     * {@inheritDoc}
+     * - Expands product table items with abstract product approval status.
+     *
+     * @api
+     *
+     * @param array<array<string, mixed>> $items
+     * @param array<array<string, mixed>> $productData
+     *
+     * @return array<array<string, mixed>>
+     */
+    public function expandTableData(array $items, array $productData): array
+    {
+        // The code below is code sample that just works. (Learn how to make it fancy in https://docs.spryker.com/docs/dg/dev/architecture/architectural-convention.html)
+        $productAbstractIds = array_map(function ($item){
+            return $item['id_product_abstract'];
+        }, $items);
+
+        $productCategoryCollection = (new ProductCategoryFacade())
```

Comment:
mb we can use AbstractPlugin functionality and fetch facade with built-in methods?
I get that this is an example, but this is already here and can be used